.github/workflows/build.yml

```diff
@@ -48,6 +48,36 @@ jobs:
           verbose: true # optional (default = false)
         if: runner.os == 'Linux'
 
+  standalone-lfc-tests:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      # Setup Build dependencies
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1.4.3
+        with:
+          # The Java version to make available on the path. Takes a whole or semver Java version, or 1.x syntax (e.g. 1.8 => Java 8.x). Early access versions can be specified in the form of e.g. 14-ea, 14.0.0-ea, or 14.0.0-ea.28
+          java-version: 11
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Run standalone LFC tests
+        run: |
+          ./gradlew :org.lflang.lfc:test
+      - name: Report to CodeCov
```

Comment:
I don't think this should be here. Only one job is supposed to report to CodeCov, and it has to be one that is driven by JUnit.